apps/frontline-widgets/src/app/messenger/components/conversation.tsx

```
@@ -161,16 +167,16 @@ export function ConversationMessage({
-                )}
-                dangerouslySetInnerHTML={{
-                  __html: DOMPurify.sanitize(content || ''),
-                }}
-              />
-            )}
+            <span
+              className={cn(
+                'truncate line-clamp-1 w-auto',
+                !isCustomerRead && 'text-accent-foreground font-bold',
+              )}
+              dangerouslySetInnerHTML={{
```

Biome (2.4.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

(lint/security/noDangerouslySetInnerHtml)